README.md

```diff
@@ -22,6 +22,7 @@ This repository is meant to help new users start using the Arm-based AWS Gravito
 * [TensorFlow](tensorflow.md)
 * [Known issues and workarounds](#known-issues-and-workarounds)
 * [AWS Managed Services available on Graviton](managed_services.md)
+* [Graviton Performance Runbook](perfrunbook/graviton_perfrunbook.md)
```

Comment:
Maybe also add a link to the runbook from Section "# Optimizing for Graviton" below.